Update case-study-guidelines.md (#1073)
cncf/foundation: policies-guidance/case-study-guidelines.md

* Product Type
* CNCF Projects Used
* Bullet point list of Metrics for “By the numbers” infographic

### What to Include ###

* Note: All case studies must demonstrate at least one tangible business impact of CNCF incubating or graduated projects
* Title & Summary
  * Use a clear, results-oriented headline
  * Example–“How [Company] Reduced Deployment Time by 70% with [CNCF Project]”
  * Include a 2–3 sentence executive summary with key outcomes
* Company Overview - briefly introduce the company: size, industry, mission, and digital strategy focus
* Challenges
  * Describe specific technical or business problems and measurable goals
* Solution–detail the solution with an emphasis on:
  * Why CNCF/cloud native tools were chosen
  * Architecture diagrams, if applicable
  * Phases of adoption (e.g., pilot → scale)
* Results
  * Quantifiable improvements (time saved, costs reduced, performance gains)
  * Developer productivity and team morale
  * Before/after comparisons or metrics
  * Key lessons learned and future plans (if applicable)
* Future Plans
  * Upcoming initiatives or deeper integration with CNCF technologies
  * Potential for community contributions or open sourcing

> * Quote–must be included 
>   * Someone who worked on or with the project - engineers, SREs, or product leads help humanize the story
>   * Mention CNCF 

Reviewer: ```suggestion
* At least one quote must be included 
  * From someone who worked on or with the project - engineers, SREs, or product leads help humanize the story 
```

## Overview ##

You’ve got great case studies. We want our ecosystem (and beyond) to hear about the impact CNCF projects and cloud native technologies are having on your business. They are also a valuable resource for organizations considering adopting cloud native technologies and helping showcase participating CNCF organizations as leaders in their field.

**There are a number of ways you can participate in the case study program. Please see further information and eligibility criteria below** 

[Written case study](https://marketingform.cncf.io)
* Open to any end user (CNCF member or non-member companies that use, but don’t sell cloud native technologies) organization working with a * CNCF incubating or graduated project 
* CNCF members can provide a case study centered on your customer use case with details about how you helped with the implementation
* All case studies must demonstrate at least one tangible business impact of CNCF incubating or graduated projects

> * NOTE: Case studies from vendor companies selling cloud native services to external organizations are not eligible

Reviewer: duplicate of above

## Ensure maximum engagement. Review the guidelines below to optimize your case study. ##

### Tips for Getting Started & Engagement ###

* Simple and concise is the way to go
* Less is more – 750 words or less is optimal, but no more than 1,000
* Bullets
  * break up long paragraphs of text 
  * help highlight key points, successes, and metrics
* Section headers
  * Clear, results-oriented headline, section headers (instead of stating Challenge, Solution, Results):
   * Add interest 
   * Provide context for the section (i.e. takeaways)
   * Help with SEO
   * Tip: ChatGPT is great for help
* Visuals and architecture diagrams (when applicable) add interest and help tell your story
  * Ensure they are hi-res
* Feature individual voices
  * Please use the active voice in CNCF case studies. More information on active vs passive voices can be [found here](https://www.grammarly.com/blog/active-vs-passive-voice/) 
* Conversational and honest: avoid jargon-heavy corporate speak
  * All CNCF publications use "open source" and "cloud native" as the proper style

>   * Projects use different styles of capitalization. Reference the [graduated and incubating project listing](https://www.cncf.io/projects/%5d) to confirm

Reviewer: ```suggestion
```

* Note: All case studies must demonstrate at least one tangible business impact of CNCF incubating or graduated projects
* Title & Summary
  * Use a clear, results-oriented headline
  * Example–“How [Company] Reduced Deployment Time by 70% with [CNCF Project]”
  * Include a 2–3 sentence executive summary with key outcomes
* Company Overview - briefly introduce the company: size, industry, mission, and digital strategy focus
* Challenges
  * Describe specific technical or business problems and measurable goals
* Solution–detail the solution with an emphasis on:
  * Why CNCF/cloud native tools were chosen
  * Architecture diagrams, if applicable
  * Phases of adoption (e.g., pilot → scale)
* Results
  * Quantifiable improvements (time saved, costs reduced, performance gains)
  * Developer productivity and team morale
  * Before/after comparisons or metrics
  * Key lessons learned and future plans (if applicable)
* Future Plans
  * Upcoming initiatives or deeper integration with CNCF technologies
  * Potential for community contributions or open sourcing
* Quote–must be included 
  * Someone who worked on or with the project - engineers, SREs, or product leads help humanize the story

>   * Mention CNCF 

Author: - Case studies can be about open sourcing proprietary software
- Include a quote in the case study from a contributor or maintainer of a project
- Mention how CNCF positively impacted their company/org

Do you think the current copy needs to be re-worded? If so, we are open to suggestions.
